### PR TITLE
Add public computer labs page

### PR DESCRIPTION
**Pull request by ecsteadman**

### CODE REVIEW

File: themes/computer-lab/layout/public-computers.jade [hunk 1]
```diff
@@ -1,40 +1,79 @@
 extends base
 
 block site_body
-  section#about
+  section#public-computers
     .container
-      h2.section-title If you need to use a computer, consider:
+      h2.section-title We currently don't have computers for public use. 
+      
+      h3.section-title Here is a list of places in New York City with public computers or wifi. Click on the links for more information.
```
Comment (ptsteadman) [inline themes/computer-lab/layout/public-computers.jade]: use WiFi instead of wifi

File: themes/computer-lab/layout/public-computers.jade [hunk 2]
```diff
@@ -1,40 +1,79 @@
...
+
+        li
           p
           a(target='_blank' href='https://foursquare.com/v/terminal-d-delta-terminal/4ace64bef964a52053d020e3') Delta terminal at Laguardia Airport
-         li
+          p 2,500 iPads available for people waiting for flights. 
+
+        li
+          p
+          a(target='_blank' href='https://www.powrplnt.org/') Digital Vans
```
Comment (ptsteadman) [inline themes/computer-lab/layout/public-computers.jade]: Wrong link

File: themes/computer-lab/layout/public-computers.jade [hunk 3]
```diff
@@ -1,40 +1,79 @@
...
+          p
+          a(target='_blank' href='http://www.fedex.com/us/office/computer-rental.html') FedEx
+          |  or 
+          a(target='_blank' href='https://www.theupsstore.com/store-services/computer-access') UPS stores
+          p Computers, printers and scanners available for a fee.
+
+        li
+          p
+          a(target='_blank' href='https://www.timeout.com/newyork/hotels/best-hotel-lobbies-with-free-wi-fi') Hotel lobbies
+          p Various hotel lobbies have free wifi that can be accessed by the public. Food and drink available also.
```
Comment (ptsteadman) [inline themes/computer-lab/layout/public-computers.jade]: Remove food and drink sentence.

File: themes/computer-lab/layout/public-computers.jade [hunk 4]
```diff
@@ -1,40 +1,79 @@
...
-          a(target='_blank' href='http://www.bestbuy.com/site/store-locator') Best Buy stores
-         li
+          a(target='_blank' href='https://www.yelp.com/search?find_desc=Coffee+Shops+Free+Wifi&find_loc=New+York%2C+NY') Coffee shops
+          p Various coffee houses and restaurants have free wifi that can be accessed by the public. Food and drink available also.
+
+        li
           p
           a(target='_blank' href='https://foursquare.com/v/terminal-d-delta-terminal/4ace64bef964a52053d020e3') Delta terminal at Laguardia Airport
-         li
+          p 2,500 iPads available for people waiting for flights.
```
Comment (ptsteadman) [inline themes/computer-lab/layout/public-computers.jade]: Let's remove this if you need a ticket.

File: themes/computer-lab/layout/public-computers.jade [hunk 5]
```diff
@@ -1,40 +1,79 @@
...
+          p
+          a(target='_blank' href='https://www.bklynlibrary.org/use-the-library/tech') Brooklyn Public Library
+          p Public computers, laptops, and wifi available for free for those with a library card. Printers, copiers, and scanners available for a fee. 
+
+        li
           p
-          a(target='_blank' href='http://www.bestbuy.com/site/store-locator') Best Buy stores
-         li
+          a(target='_blank' href='https://www.yelp.com/search?find_desc=Coffee+Shops+Free+Wifi&find_loc=New+York%2C+NY') Coffee shops
+          p Various coffee houses and restaurants have free wifi that can be accessed by the public. Food and drink available also.
```
Comment (ptsteadman) [inline themes/computer-lab/layout/public-computers.jade]: WiFi

Comment (ptsteadman) [inline themes/computer-lab/layout/public-computers.jade]: Remove 'Food and drink..'

File: themes/computer-lab/layout/public-computers.jade [hunk 6]
```diff
@@ -1,40 +1,79 @@
...
+          p
+          a(target='_blank' href='https://www.timeout.com/newyork/hotels/best-hotel-lobbies-with-free-wi-fi') Hotel lobbies
+          p Various hotel lobbies have free wifi that can be accessed by the public. Food and drink available also.
+
+        li
           p
-          a(target='_blank' href='https://www.timeout.com/newyork/hotels/best-hotel-lobbies-with-free-wi-fi') Hotel Lobbies
-         li
+          a(target='_blank' href='https://www.nycgovparks.org/crc') New York City Parks Computer Resource Centers
+          p Computer classes, internet access, professional software and training available. Membership required.
```
Comment (ptsteadman) [inline themes/computer-lab/layout/public-computers.jade]: Make a link to membership.

File: themes/computer-lab/layout/public-computers.jade [hunk 7]
```diff
@@ -1,40 +1,79 @@
...
+
+        li
           p
-          a(target='_blank' href='https://www.ups.com/dropoff') UPS* 
-          a(target='_blank' href='https://www.fedex.com/locate/') or FedEx stores*
+          a(target='_blank' href='https://www.powrplnt.org/') POWRPLNT
+          p Gallery space, internet lounge, and computers for people interested in digital art. Classes available as well.
+
       ul
+        p.more If you know of any other locations with public computers or wifi, please email us: hi@computerlab.io.
```
Comment (ptsteadman) [inline themes/computer-lab/layout/public-computers.jade]: remove wifi, please email us at

**Comment by ptsteadman** [comment]
- Justify bottom copy with top copy.
- Rank by how good of an option you think each is.
- More info about memberships.
- Add that some hotel lobbies might have computers.
- Test different device sizes.

**Comment by ptsteadman** [comment]
- In bottom text, also say "or make a pull request here to edit this website", link to this github repo 'here'

**Comment by r1b** [comment]
Looks good!

**Pull request merged**